src/luma/deploy.py

```diff
@@ -144,7 +144,7 @@ def monitor_deployment(package_name: str):
         deployment_id: The id of the deployment to monitor
         package_name: The name of the package being deployed
     """
-    logger.info("Monitoring deployment...")
+    logger.debug("Monitoring deployment...")
```

Comment:
I don't think users will see these logs by default anymore. Is that the desired behavior?